lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -713,21 +690,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :dynamic_name => true,
           :description => 'A comma separated list of status codes, possibly including ranges. Errors associated with these status codes, where applicable, will be treated as expected.'
         },
-        :'error_collector.ignore_errors' => {
-          :default => 'ActionController::RoutingError,Sinatra::NotFound',
-          :public => true,
-          :type => String,
-          :deprecated => true,
-          :allowed_from_server => true,
-          :dynamic_name => true,
-          :description => <<~DESCRIPTION
-            Use `error_collector.ignore_classes` instead. Specify a comma-delimited list of error classes that the agent should ignore.
 
-                <Callout variant="caution">
-                  Server side configuration takes precedence for this setting over all environment configurations. This differs from all other configuration settings where environment variable take precedence over server side configuration.
-                </Callout>
-          DESCRIPTION
```

Comment:
Does `error_collector.ignore_classes` have a similar SSC priority that we should call out in the description of `error_collector.ignore_classes?

Reply:
No, that shouldn't affect ignore_classes. The only config source higher than server side config is env vars, but ignore_classes is not accepted via env vars, so server side config is highest precedence anyways

Reply:
Awesome, thanks!